README.md

```diff
@@ -23,24 +23,24 @@ please use [alluxiofs](https://github.com/fsspec/alluxiofs) instead.
 
 Install from source
 ```
-cd alluxio-python-library
+cd alluxio
 python setup.py sdist bdist_wheel
-pip install dist/alluxio_python_library-0.1-py3-none-any.whl
+pip install dist/alluxio-<alluxio_client_version>-py3-none-any.whl
 ```
 
 ## Usage
 
 ### Initialization
-Import and initialize the `AlluxioFileSystem` class:
+Import and initialize the `AlluxioClient` class:
 ```
 # Minimum setup for Alluxio with ETCD membership service
-alluxio = AlluxioFileSystem(etcd_hosts="localhost")
+alluxio = AlluxioClient(etcd_hosts="localhost")
```

Comment:
Could you update it to `alluxio_client`? I remember we have used the name `alluxio` in the fsspec example.